apps/web/components/Proposals.tsx

```diff
@@ -210,44 +211,49 @@ export function Proposals({
     useTransactionNotification(allocateData);
 
   useEffect(() => {
-    triggerRenderProposals();
+    // triggerRenderProposals();
+    runIsMemberQuery();
   }, [txConfirmationHash]);
 
   useEffect(() => {
     updateTransactionStatus(allocateStatus);
   }, [allocateStatus]);
 
   const submit = async () => {
-    const encodedData = getEncodedProposals(inputs, stakedFilters);
+    const proposalsDifferencesArr = getProposalsInputsDifferences(
+      inputs,
+      stakedFilters,
+    );
+    const encodedData = encodeFunctionParams(cvStrategyABI, "supportProposal", [
+      proposalsDifferencesArr,
+    ]);
     const poolId = Number(strategy.poolId);
     writeAllocate({
       args: [BigInt(poolId), encodedData as AddressType],
     });
   };
 
-  const getEncodedProposals = (
+  // this calculations breaks when dealing with < 50 wei
+  const getProposalsInputsDifferences = (
     inputData: ProposalInputItem[],
     currentData: ProposalInputItem[],
   ) => {
     const resultArr: [number, BigInt][] = [];
     inputData.forEach((input) => {
       let row: [number, bigint] | undefined = undefined;
-      if (input.value > 0) row = [Number(input.id), BigInt(input.value)];
+      if (input.value > 0)
+        row = [Number(input.id), BigInt(Math.round(input.value))];
       currentData.forEach((current) => {
         if (input.id === current.id) {
-          const dif = BigInt(input.value - current.value);
+          const dif = BigInt(Math.round(input.value) - current.value);
           row = [Number(input.id), dif];
         }
       });
-      if (!!row) resultArr.push(row);
+      if (!!row && row[1] !== 0n) resultArr.push(row);
```

Comment:
`!!row` its unecessary there, just use `if (row &&)`

Reply:
you are right, now that i see it is redundant.